start-from-zero/evaluation-zero.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Evaluation</title>
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/7.0.0/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    
+    <nav class="menu">
+        
+         <div class="container" id='height-100'>
```

Comment:
I use Emmet plugin on my IDE(He's the one who's using double quotes) :) . Most of the times I don't hard code every piece of code from my HTML files. In this particular case I wrote nav.menu>.container, and that would do the trick. After some time I decided that I needed to add an ID to that div and for that I used my own "coding conventions". I will stick to the double quotes from now on. Thanks for pointing out.